### PR TITLE
Update customize-the-app-launcher.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to the navigation

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7020

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/admin/manage/customize-the-app-launcher.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ You can add your own custom tiles to the app launcher that point to SharePoint s
 
 1. In the admin center, go to the **Settings** > **Org Settings** > **Organization profile** tab.
     
-2. On the **Organization profile** page, next to   **Add custom tiles for your organization**, select **Edit**.
+2. On the **Organization profile** page, go to **Custom App launcher tiles**,  If you select the three dots next to your **Custom Tile** and Select **Edit custom tile**.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/customize-the-app-launcher.md:70]:
```suggestion
2. On the **Organization profile** page, go to **Custom app launcher tiles**. Select the three dots next to your custom tile and then select **Edit custom tile**.
```

**Comment by yogkumgit** [comment]
@msbemba

**Pull request merged**